models/metrics.go

```diff
@@ -529,25 +527,66 @@ func (m *Metrics) SelectQueryExamples(ctx context.Context, periodStartFrom, peri
 }
 
 const queryObjectDetailsLabelsTmpl = `
-	SELECT server, database, schema, username, client_host, labels.key AS lkey, labels.value AS lvalue
-	  FROM metrics
-	 ARRAY JOIN labels
-	 WHERE period_start >= ? AND period_start <= ?
-	 {{ if index . "filter" }} AND {{ index . "group" }} = '{{ index . "filter" }}' {{ end }}
-	 ORDER BY server, database, schema, username, client_host, labels.key, labels.value
+	SELECT
+	service_name,
+	database,
+	schema,
+	username,
+	client_host,
+	replication_set,
+	cluster,
+	service_type,
+	service_id,
+	environment,
+	az,
+	region,
+	node_model,
+	node_id,
+	node_name,
+	node_type,
+	machine_id,
+	container_name,
+	container_id,
+	agent_id,
+	agent_type,
+	labels.key AS lkey,
+	labels.value AS lvalue
+	FROM metrics
+	ARRAY JOIN labels
+	WHERE period_start >= ? AND period_start <= ?
+	{{ if index . "filter" }} AND {{ index . "group" }} = '{{ index . "filter" }}' {{ end }}
```

Comment:
What if "filter" contains `'`? Will we have SQL injection?

Reply:
Fixed.